### PR TITLE
Get data from local datastore instead of from mock JSON

### PR DESCRIPTION
**Pull request by racheljay**
This PR adds logic to populate temporary fake data into the local environment datastore. Previously it has been pulling mock data from a static JSON file. It is now pulling from and processing data from a mock data set in the local version of the datastore.

### CODE REVIEW

File: util/populate_dev_data.go [hunk 1]
```diff
@@ -318,3 +321,152 @@ func FetchPendingRuns(wptdHost string) ([]shared.PendingTestRun, error) {
 	err := shared.FetchJSON(url, &pendingRuns)
 	return pendingRuns, err
 }
+
+func addFakeHistoryData(store shared.Datastore) {
+	// browser_name,browser_version,date,test_name,subtest_name,status
```
Comment (DanielRyanSmith) [inline util/populate_dev_data.go:326]: This looks fine as a temporary change. Can we add a TODO comment here to import real data here when staging is populated with real data?

**Review by DanielRyanSmith — COMMENTED** [review]
NIce work with taking the raw db format and converting it to be ingested client side 👍
It looks like this change breaks the timeline charts using fake data on staging/prod. Can we add a change to submit the old mocked JSON if no history is pulled from the database?

**Review by BertsLifee — APPROVED** [review]

**Review by BertsLifee — APPROVED** [review]

**Review by BertsLifee — APPROVED** [review]

**Review by DanielRyanSmith — APPROVED** [review]
Looks great! This should work just as intended once the raw data exists in staging 🙂

**Comment by KyleJu** [comment]
WOOHOO Great work!  Don't forget to add tests for the `testHistoryHandler` function!

**Pull request merged**